specs/src/specs/params.md

```diff
@@ -64,3 +64,7 @@ are blocked by the `x/paramfilter` module.
 | staking.MaxValidators                         | 100                                         | Maximum number of validators.                                                                                                                                                                   | False                     |
 | staking.MinCommissionRate                     | 0.05 (5%)                                   | Minimum commission rate used by all validators.                                                                                                                                                 | True                      |
 | staking.UnbondingTime                         | 1814400 (21 days)                           | Duration of time for unbonding in seconds.                                                                                                                                                      | False                     |
+
+### Tests
+See [/x/paramfilter/test/governance_params_test.go](/x/paramfilter/test/governance_params_test.go) for tests that attempt to modify a parameter.
+Note that `TimeIotaMs` is not in `conensus.block`, so a governance proposal can not be submitted to modify it. Additionally, the `x/mint` module does not have any governance modifiable params as justified in [/x/mint/README.md#params](/x/mint/README.md#params)
```

Comment:
[question] if `TimeIotaMs` is not in `consensus.block`, should it be removed from the list of params above? Do you know if this field exists elsewhere?

Reply:
For context, `ConsensusParams` in `celestia-core/abci/types/types.pb.go` as seen [here](https://github.com/celestiaorg/celestia-core/blob/e67de4517780e8db93c74d8eec4e9c66edffbc99/abci/types/types.pb.go#L2809) uses `BlockParam` from `celestia-core/abci/types/types.pb.go` instead of `types1.BlockParams` from `celestia-core/proto/tendermint/types/params.pb.go`.

This version of `BlockParam` does not contain `TimeIotaMs`, however I am not sure why `types1.BlockParams` isn't used instead, or if `ConsensusParams` from `celestia-core/proto/tendermint/types/types.pb.go` is another valid alternative that could be used instead in `baseapp.go`. 

With this context in mind and knowing the field exists somewhere else (but seems to not be used), I think it would be good to clarify if the intended version of `BlockParams` is being used in the first place before it is removed from `params.md`. Do you have any thoughts on this or know of anyone else who might?

Reply:
For further context on why I ask this, it is because I use `BaseApp.GetConsensusParams()` when testing if the governance modification was successful. [This commit here](https://github.com/celestiaorg/celestia-app/pull/2919/commits/39fede889d8cf6504427c1df639792fae1994764) shows this code.